packages/terra-overlay/src/terra-dev-site/test/overlay/LoadingOverlay/CustomMessageLoadingOverlay.test.jsx

```diff
@@ -1,4 +1,24 @@
 import React from 'react';
 import LoadingOverlay from '../../../../LoadingOverlay';
 
-export default () => <LoadingOverlay isOpen message="Custom Loading Message" id="terra-LoadingOverlay" />;
+class CustomMessageLoadingOverlay extends React.Component {
+  constructor() {
+    super();
+
+    this.state = { show: true };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ show: false });
+    }, 5000);
+  }
```

Comment:
We discussed this issue in scrum earlier this week, I believe we want to avoid timers in the test examples. We may want to update this example to use the Container as well and remove the timeout.

cc: @bjankord

Reply:
I was made aware that we had to use "Overlay Containers" and not timeout as per discussion on #2476 . I also implemented the same as can be seen in https://github.com/cerner/terra-core/pull/2481/commits/28471e882788407dd7b73e2f8574487a92cec06f commit.
But, there is a test in `loading-overlay-spec.js` to check whether the loading overlay is in "fullscreen mode" or not. This check was added to test the following functionality:
https://github.com/cerner/terra-core/blob/master/packages/terra-overlay/src/Overlay.jsx#L151.
So i changed from using an "Overlay Container" to a "timer" for that test.

Is there any other approach for this while keeping that check?
**'or'**
Do I go ahead with an "Overlay Conainer" by removing that check?

Reply:
I don't have a strong preference on this one way or another. @mjhenkes @ryanthemanuel @tbiethman Thoughts?

Reply:
I'm fine with the timer on the test page.